unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/PlacesSubSection/PlaceCard/PlaceCardComponentView.cs

```diff
@@ -182,10 +191,38 @@ public override void RefreshControl()
         SetPlaceAuthor(model.placeAuthor);
         SetNumberOfUsers(model.numberOfUsers);
         SetCoords(model.coords);
+        SetFavoriteButton();
+
+        //Temporary untill the release of the functionality
+        if (!DataStore.i.HUDs.enableFavoritePlaces.Get())
```

Comment:
Avoid static usages in views, this can be injected by the controller or whoever instances this view.

Reply:
it's a bit problematic due to the extensive usage of the view in many different places where not everywhere would be easy to inject the dependency, I thought of doing the ugly static usage because it will disappear pretty soon, but if we want I can do the DI everywhere

Reply:
okay fair enough!